modules/programs/autorandr.nix

```diff
@@ -79,6 +79,13 @@ let
         default = null;
         example = "left";
       };
+
+      transform = mkOption {
+        type = types.str;
```

Comment:
Maybe it's better to have it as a list of strings? List of floats would be even better, but I guess we might have some weird problems with binary/decimal representations?

Reply:
I think @uvNikita meant to use a list of strings or floats for prettification. Could go further and define a matrix type. I.e., maybe have

```nix
let

  matrixOf = with types;
    let
      list3 = elemType: addCheck (listOf elemType) (l: length l != 3);
    in
      uniq (list3 (list3 float))
      // { description = "3×3 matrix of floating point numbers"; };
```

at the top of the module and use that type here? The example should then become

```nix
[
  [ 0.6 0.0 0.0 ]
  [ 0.0 0.6 0.0 ]
  [ 0.0 0.0 1.0 ]
]
```

and

```nix
lib.concatMapStringsSep "," toString (lib.flatten config.transform)
# Evaluates to
"0.600000,0.000000,0.000000,0.000000,0.600000,0.000000,0.000000,0.000000,1.000000"
```

Note everything above is dry-coded so there might be mistakes.

Reply:
@rycee @uvNikita I have updated the PR, thanks for forcing me to learn types! TIL!